podcasts/MainTabBarController.swift

```diff
@@ -556,6 +556,10 @@ class MainTabBarController: UITabBarController, NavigationProtocol {
         tabBar.scrollEdgeAppearance = appearance
         tabBar.unselectedItemTintColor = AppTheme.unselectedTabBarItemColor()
         tabBar.tintColor = AppTheme.tabBarItemTintColor()
+        
+        // Link userInterfaceStyle to Theme type so iOS's Increase Contrast setting does the right thing
+        let isDarkTheme = Theme.isDarkTheme()
+        tabBar.overrideUserInterfaceStyle = isDarkTheme ? .dark : .light
```

Comment:
I believe we can rewrite that in a single line while keeping the readability:

```suggestion
        tabBar.overrideUserInterfaceStyle = Theme.isDarkTheme() ? .dark : .light
```

Reply:
Yes, good call. That's a remnant of more complicated code I had tried. I should have removed it since I ended up with the single ternary. Thanks!